xds/src/main/java/io/grpc/xds/EdsLoadBalancer.java

```diff
@@ -328,10 +368,17 @@ public void shutdown() {
 
     @Override
     public void onEndpointChanged(EndpointUpdate endpointUpdate) {
-      channelLogger.log(
-          ChannelLogLevel.DEBUG,
-          "EDS load balancer received an endpoint update: {0}",
-          endpointUpdate);
+      logger.log(XdsLogLevel.DEBUG, "Received endpoint update: {0}", endpointUpdate);
+      if (logger.isLoggable(XdsLogLevel.INFO)) {
+        logger.log(
```

Comment:
Duplicate logging?

Reply:
`EndpointUpdate` contains complex fields, printing out the whole object in INFO level seems to be too verbose. 

Yes, in some places we tend to do both INFO and DEBUG logging. INFO log will let you briefly gauge what's happening and DEBUG log is helpful for debugging the implementation. I would expect normal users (i.e., people do not read the actual code) to look at INFO log for the most and implementors to look at DEBUG logs for finding out issues. That's why I would like to make INFO logs natural and reader-friendly, avoid printing out implementation details.